src/main/java/gregtech/loaders/oreprocessing/MaterialRecipeHandler.java

```diff
@@ -362,6 +362,24 @@ public static void processFrame(OrePrefix framePrefix, SolidMaterial material) {
         }
     }
 
+    public static void processMetalCasing(OrePrefix prefix, SolidMaterial material) {
+        if (material.hasFlag(GENERATE_PLATE | GENERATE_FRAME | GENERATE_METAL_CASING)) {
```

Comment:
that's not how things work. reason of these if checks is protecting handling code from items registered by other mods, not preventing fucked up frames from gems from generating recipes.

Reply:
I'm not understanding your comment, what I'm supposed to do?